internal/pkg/describe/lb_web_service.go

```diff
@@ -287,6 +287,7 @@ func underline(headings []string) []string {
 type WebServiceRoute struct {
 	Environment string `json:"environment"`
 	URL         string `json:"url"`
+	Public      *bool  `json:"public,omitempty"`
```

Comment:
Do we have three different states for this field? If not it can probably be `bool`?

Reply:
We do have a third - since LBWS uses the same object. I didn't want to add a new field to lbws since it's inherently "public"

Reply:
What if this was an `enum`? where the type instead of a `bool` is a `string` and the field name is `"ingress"`?

I'm thinking of a scenario where maybe the url is accessible within an environment vs. a shared VPC vs. public